src/transform/xsl/default.xsl

```diff
@@ -430,18 +430,18 @@
     </xsl:template>
 
     <xsl:template match="fn-group/fn">
-        <li>
+        <xsl:variable name="fn-number">
+            <xsl:number level="any" count="fn[not(ancestor::front)]" from="article | sub-article | response"/>
+        </xsl:variable>
+        <li id="fn{$fn-number}">
+            <span class="footnotemarker" id="n{$fn-number}"></span>
```

Comment:
Doesn't this lead to two DOM elements having identical IDs?

Reply:
On is `fn{id}` and ther other is `n{id}` the link in the text uses `nm{id}`. I think we added the n{id} one as other stuff was using the `fn{id}` one and we didn't want to mess around with it.

Reply:
sounds good